### PR TITLE
Add Open Compute Project to the landscape

### PR DESCRIPTION
**Pull request by catblade**
Add Open Compute Project, an organization, to the landscape, per request from Ed Vielmetti.

### CODE REVIEW

File: website/content/about/SustainabilityUseCasesAndLandscape2023.md [hunk 1]
```diff
@@ -427,6 +427,8 @@ There are a number of sustainability initiatives ongoing, if we've missed one pl
...
   Get to know the carbon footprint of your cloud usage - and reduce it.<br>
 Cloud Carbon Footprint is an open source tool that provides visibility and tooling to measure, monitor and reduce your cloud carbon emissions. We use best practice methodologies to convert cloud utilization into estimated energy usage and carbon emissions, producing metrics and carbon savings estimates that can be shared with employees, investors, and other stakeholders.
+* 🐝 [Open Compute Project](https://www.opencompute.org/projects/heat-reuse)<br>
+  Almost 100% of the energy used in a processor turns into heat. Up until very recently this has been a liability and a challenge, immense quantity of heat needed to be removed employing even more energy for it. With the proper design, data center cooling systems can be converted into a heat source and a heat management opportunity that in turn converts the heat from a liability to an asset. The OCP Heat Reuse subgroup explores these challenges and opportunities. Its goal is to suggest solutions to allow easier implementations of technologies aimed at harnessing the heat and turn it into profit instead of a cost.
```
Comment (cmcantalupo) [inline website/content/about/SustainabilityUseCasesAndLandscape2023.md:431]:
```suggestion
  Almost 100% of the energy used in a processor turns into heat. Up until very recently this has been a liability and a challenge: an immense quantity of heat to be removed which requires a significant additional energy expenditure. With the proper design, data center cooling systems can be converted into a heat source and a heat management opportunity that in turn converts the heat from a liability to an asset. The OCP Heat Reuse subgroup explores these challenges and opportunities. Its goal is to suggest solutions to enable implementations of technologies designed to harness the heat to turn cost into profit.```

Comment (cmcantalupo) [reply website/content/about/SustainabilityUseCasesAndLandscape2023.md:431]: Small wording changes, take whatever seems good.

**Comment by leonardpahlke** [comment]
Before merging this, lets discuss briefly how we intent to version the document. See https://github.com/cncf/tag-env-sustainability/issues/115

**Comment by catblade** [comment]
> Before merging this, lets discuss briefly how we intent to version the document. See #115

I'm fine with that.  Let's discuss Wed?

**Review by vielmetti — APPROVED** [review]
Looks like a good summary!

**Pull request merged**